### DISCUSSION TITLE
Populate board and get AI response

### DISCUSSION THREAD

**Discussion by ataylorappt** [lorserker/ben#85] [General]
Hello

Is there a way to set up a game mid way through play and then get a response from the AI on what move it would do next ?

If not would I be able to get details on how I could possibly implement this feature into the engine ?

I would like to circumvent using the table monitor as its windows specific and able to get AI responses for multiple games on the fly if I just supply the engine with the game state.

Thanks

**Comment by ThorvaldAagaard** [comment]
There are some examples that you might be able to use as a starting point.
Look at CardByCardAnalysis.ipynb

I have implemented, that you can feed a board including bidding, but I haven't done anything to play half-way thru the deal.
